### ISSUE TITLE
Various items do not scale for Low Vision Users

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31763]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

-	Colour picker: this sub menu (accessed whenever the user tries to alter the colour of an element) has tiny icons and the text size doesn’t scale up to match the user’s preferred text size.

-	Tooltips/icon label text is not scalable. In iOS this can be solved using Hovertext, which will produce a label in the user’s preferred size, but it is slow to respond and there is no equivalent accessibility feature in Windows.

-	Icon size: the graphic icons within the palette submenus currently remain the same size, regardless of the font size selected, whilst the icons in the note editor toolbar scale up. The time and key signatures are especially hard to read. This is somewhat solvable in iOS if the user has Hovertext enabled but see the previous note re response time and operating system.

-	Mixer palette: It’s hard to set the levels as the sliders and numbers are very small. These either need to be made bigger, or an alternative method is needed.
 
-	The scroll bars throughout the software are tiny and hard to find.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.5.2

### Regression

I was unable to check

### Operating system

iOS: Sequoia 15.6

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
